assets/js/blocks/checkout/index.tsx

```diff
@@ -24,25 +23,20 @@ const settings = {
 			/>
 		),
 	},
-	category: 'woocommerce',
-	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
-	description: __(
-		'Display a checkout form so your customers can submit orders.',
-		'woo-gutenberg-products-block'
-	),
-	supports: {
-		align: [ 'wide' ],
-		html: false,
-		multiple: false,
+	attributes: {
+		...metadata.attributes,
+		...blockAttributes,
+		...deprecatedAttributes,
 	},
-	attributes: blockAttributes,
-	apiVersion: 2,
 	edit: Edit,
 	save: Save,
 	// Migrates v1 to v2 checkout.
 	deprecated: [
 		{
-			attributes: blockAttributes,
+			attributes: {
+				...metadata.attributes,
+				...blockAttributes,
```

Comment:
Just checking; we missing deprecatedAttributes here?

Reply:
Yup, good catch! Thanks Mike 😓